tests/conftest.py

```diff
@@ -16,8 +16,10 @@
 from baybe.acquisition import qExpectedImprovement
```

Comment:
aren't hypothesis tests missing for the new constraints?

Reply:
Ok, yes. But how about we merge now and let @Waschenbacher add them in a separate PR? I think that would allow us to move faster, because the next cardinality constraint is already incoming and I think she needs to get familiar with hypothesis first

Reply:
fine with me as long as its not forgotten

Reply:
can't be forgotten, we still need to finish our hypothesis hierarchy. The lastest when we intergrate search spaces we'll have to do it 🙃

Reply:
Sure! Will add the hypothesis test for the cardinality constraints in a separate PR.